### ISSUE TITLE
Error when config file is not published

### ISSUE THREAD

**Issue by jameswagoner** [illuminatech/data-provider#4]
### What steps will reproduce the problem?
1. Install via composer
2. Build a data provider from examples
3. Receive errors

### What is the expected result?
A json response with query results

### What do you get instead?
```
#message: "array_replace_recursive(): Argument #2 must be of type array, null given"
#code: 0
#file: "/app/vendor/illuminatech/data-provider/src/DataProvider.php"
#line: 123
```

### Additional info

| Q                         | A
| ------------------------- | ---
| This Package Version      | 1.0.4
| Laravel Framework Version | Latest
| PHP version               | 8.2
| Operating system          | Docker

**Comment by jameswagoner** [comment]
Investigated further and seems issue happens when there is no published config or config option present in `DataProvider::new()` method

**Comment by klimov-paul** [comment]
Resolved by commit e06d873487199c7bb6c600d25bfd09740a4c9b8b
